src/models/document/base-document-content.ts

```diff
@@ -686,7 +688,7 @@ export const BaseDocumentContentModel = RowList.named("BaseDocumentContent")
     }
   }))
   .actions((self) => ({
-    copyTilesIntoExistingRow(tiles: IDropTileItem[], rowInfo: IDropRowInfo) {
+    copyTilesIntoExistingRow(tiles: IDropTileItem[], rowInfo: IDropRowInfo, makeTitlesUnique: boolean) {
```

Comment:
No change needed but I'm not a big fan of passing raw booleans when the parameter + function name isn't self-documenting.  I think passing an options object of type `{makeTitlesUnique?: boolean}` would make the code more self-documenting at the call site.